### PR TITLE
Add reuseable_alert, reuseable_alert_option & reuseable_monitoring_option

### PR DESCRIPTION
**Pull request by nurRiyad**
Add reuseable_alert, reuseable_alert_option & reuseable_monitoring_option to Elasticsearch, MariaDB, MongoDB, MySQL, Postgres & Redis

Signed-off-by: AlAsadNurRiyad <asadnurriyad@appscode.com>

### CODE REVIEW

File: charts/uibytebuildersdev-component-alert-option/ui/language.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+en:
+  labels:
+    label:
+      key: Key
+      label: Label
+      value: Value 
+    alert_enable_question: Enable Alert?
+    alert_profile: Alert Profile
+
```
Comment (shakib779) [inline charts/uibytebuildersdev-component-alert-option/ui/language.yaml:9]: sort all the keys

File: charts/kubedbcom-redis-editor/ui/create-ui.yaml [hunk 2]
```diff
@@ -111,6 +111,19 @@ steps:
...
+      chart:
+        name: uibytebuildersdev-component-alert
+        version: v0.4.3
+      moduleResolver: fetchJsons
+      schema:
+        $ref: schema#/properties/form/properties/alert
+      type: reusable-element
+    type: single-step-form
+  id: alert
+  title: Alert
```
Comment (shakib779) [inline charts/kubedbcom-redis-editor/ui/create-ui.yaml:126]: Read it from language json

File: charts/kubedbcom-mysql-editor-options/ui/functions.js [hunk 3]
```diff
@@ -479,5 +517,6 @@ return {
 	getMachineListForOptions,
 	setResourceLimit,
 	setLimitsCpuOrMem,
-	setMachineToCustom
+	setMachineToCustom,
+  updateAgentValue,
```
Comment (shakib779) [inline charts/kubedbcom-mysql-editor-options/ui/functions.js:521]: Fix indentation

**Review by shakib779 — COMMENTED** [review]
perform the requested changes

File: charts/uibytebuildersdev-component-alert-option/ui/language.yaml (hunk 1 above)
Comment (nurRiyad) [reply charts/uibytebuildersdev-component-alert-option/ui/language.yaml:9]: done

File: charts/kubedbcom-redis-editor/ui/create-ui.yaml (hunk 2 above)
Comment (nurRiyad) [reply charts/kubedbcom-redis-editor/ui/create-ui.yaml:126]: Done

File: charts/kubedbcom-mysql-editor-options/ui/functions.js (hunk 3 above)
Comment (nurRiyad) [reply charts/kubedbcom-mysql-editor-options/ui/functions.js:521]: Done

**Review by shakib779 — APPROVED** [review]
LGTM

**Pull request merged**